### DISCUSSION TITLE
project submission tab is not showing save and submit button

### DISCUSSION THREAD

**Discussion by Impavan007** [github/india#823] [General]
hello @EYmentor1 submission tab is not showing the save and submit button provide assistance

**Comment by ey-mentor1** [comment]
Is your project under evaluation? If yes, Can you please check now.

**Comment by ey-mentor1** [comment]
Hello,
The last date for the submission is expired. Due to that reason you are be getting this issue.
Backend team will be sharing the emails to the eligible students as per batch there batch. Once you receive the mail, post that only you can save and submit the project.
Thank you
